presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestOrcStorageManager.java

```diff
@@ -470,12 +474,22 @@ public static OrcStorageManager createOrcStorageManager(
             Optional<BackupStore> backupStore,
             ShardRecoveryManager recoveryManager)
     {
-        return createOrcStorageManager(storageService, backupStore, recoveryManager, MAX_SHARD_ROWS, MAX_FILE_SIZE);
+        return createOrcStorageManager(storageService, backupStore, new BackgroundShardBackupManager(Optional.empty(), 1), recoveryManager, MAX_SHARD_ROWS, MAX_FILE_SIZE);
```

Comment:
This seems wrong. Shouldn't it create the backup manager with the passed in backup store?